README.rst

```diff
@@ -32,11 +32,15 @@ Building the code
 
 Follow the usual CMake build workflow:
 
-* Configure the project, specify your compiler (e.g. ``gfortran``), the install
+* Configure the project, specify your compiler (e.g. ``gfortran``, ``ifort``, etc), the install
   location (e.g. ``$HOME/opt/skprogs``) and the build directory
   (e.g. ``_build``)::
 
-    FC=gfortran cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/skprogs -B _build .
+    FC=gfortran cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/skprogs -DCMAKE_Fortran_FLAGS=-fopenmp -B _build .
+
+  or::
+
+    FC=ifort cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/skprogs -DCMAKE_Fortran_FLAGS=-qopenmp -B _build .
```

Comment:
I think, this 2nd example is superfluous. (SkProgs should work with any standard conforming compiler, gfortran is just an example.)